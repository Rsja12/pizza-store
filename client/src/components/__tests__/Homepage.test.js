import React from 'react';
import Homepage from '../Homepage';
import { shallow } from 'enzyme';

it('renders Homepage component', () => {
    expect(shallow(<Homepage />)).toMatchSnapshot();
});
