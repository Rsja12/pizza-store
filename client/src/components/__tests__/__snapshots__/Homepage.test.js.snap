// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Homepage component 1`] = `ShallowWrapper {}`;
